packages/toolchain/collection.yaml

```
@@ -21,22 +21,32 @@ packages:
       autobump.revdeps: "true"
   - name: "luet-makeiso"
     category: "toolchain"
-    version: 0.3.8-15
+    version: 0.3.8-16
     upx: false
     fips: false
     labels:
       github.repo: "luet-makeiso"
       github.owner: "mudler"
       autobump.revdeps: "true"
+    requires:
+    - name: "luet"
```

```suggestion
    - name: "luet" &luet
```
Not tested :sweat_smile: